api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
```

Comment:
We usually don't add a description at the interface level (it is more for subfeature that are neither interfaces, methods nor properties.

Reply:
Fixed in https://github.com/mdn/browser-compat-data/pull/623/commits/595325849e0d2ea917cf33edb04b04e288078c43